components/brave_wallet/browser/android_page_appearing_browsertest.cc

```diff
@@ -200,9 +199,6 @@ class AndroidPageAppearingBrowserTest : public PlatformBrowserTest {
   AndroidPageAppearingBrowserTest() {
     factory_ = std::make_unique<TestWebUIControllerFactory>(kWalletPageHost);
     content::WebUIControllerFactory::RegisterFactory(factory_.get());
-    scoped_feature_list_.InitWithFeatures(
-        {}, {features::kBraveWalletFilecoinFeature,
-             features::kBraveWalletSolanaFeature});
   }
```

Comment:
These were actually disabled features for android test for some reason

Reply:
Yea I was unsure why they have to be disabled but I didn't see any CI failure so I assume the test is still working fine.
cc @vadimstruts to confirm if there's any particular reason here.

Reply:
It was disabled to prevent printing extra exceptions on the browser console.  
Resolving of that exceptions was not required to make sure that test is OK (page is shown up). 
But now seems test  should work correctly without that piece of code.